app/views/devise/shared/_links.html.erb

```diff
@@ -1,13 +1,13 @@
 <%- if controller_name != 'sessions' %>
-  <%= link_to "Log in", new_session_path(resource_name) %><br />
+  <%= link_to "Log in", new_session_path(resource_name), class: "login-btn" %><br />
```

Comment:
you can rename these classes to the same one: ui-btn or form-btn or something like that, because in the scss the styles are repeated, then just add these styles to 1 class, for the rest looks great!